.eslintrc.js

```diff
@@ -135,7 +135,7 @@ module.exports = {
 		'no-throw-literal': 'error',
 		'no-undef-init': 'error',
 		'no-undefined': 'off',
-		'no-underscore-dangle': 'error',
+		'no-underscore-dangle': 'off',
```

Comment:
Is this only for trailing? So far (on mobile) haven't seen anything that would require this change?